litgpt/eval/evaluate.py

```diff
@@ -90,15 +81,15 @@ def convert_and_evaluate(
     if not model_path.exists() or force_conversion:
         convert_lit_checkpoint(checkpoint_dir=checkpoint_dir, output_dir=out_dir)
 
-    safetensors_path = out_dir / "model.safetensors"
-    if not safetensors_path.exists() or force_conversion:
-        save_safetensors(out_dir, repo_id)
+    from lm_eval.models.huggingface import HFLM
```

Comment:
Does this work for you? I tried it and got a 

```
AttributeError: 'BaseModelOutputWithPast' object has no attribute 'logits'
```

Btw how are you providing the tokenizer with this method?

Reply:
The test seems to be passing in CI, which is what I ran. What are you running?

HFLM exposes it: https://github.com/EleutherAI/lm-evaluation-harness/blob/main/lm_eval/models/huggingface.py#L85-L91

But that would be a preexisting question. The previous version also doesn't provide a tokenizer and relies on the one loaded by HF automatically